src/Microsoft.Health.TaskManagement/RebuildIndex.cs

```diff
@@ -0,0 +1,18 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.TaskManagement
+{
+    public class RebuildIndex
+    {
+        public string TableName { get; set; }
+
+        public string IndexName { get; set; }
+
+        public string Command { get; set; }
+
+        public int Pid { get; set; }
```

Comment:
>public int Pid { get; set; } [](http://example.com/codeflow?start=8&length=28)

Please clarify the need for this.

Reply:
It means partitionId, I will modify it in next commit.